File: 0075-sort-colors/0075-sort-colors.cpp
class Solution {
public:
    void sortColors(vector<int>& nums) {
        int zeros=0;
        int ones=0;
        int twos=0;
        
        for(int i=0;i<nums.size();i++){
            if(nums[i]==0)zeros++;
            else if(nums[i]==1)ones++;
            else{
                twos++;
            }
        }
        int j=0;
        for(int i=0;i<zeros;i++){
            nums[j]=0;
            j++;
        }
        for(int i=0;i<ones;i++){
            nums[j]=1;
            j++;
        }
        for(int i=0;i<twos;i++){
            nums[j]=2;
            j++;
        }
    }
};

File: 0075-sort-colors/0075-sort-colors.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        i=0
        u=0
        j=len(nums)-1
        while u<=j:
            if nums[u]==0:
                nums[i],nums[u]=nums[u],nums[i]
                i+=1
                u+=1
            elif nums[u]==2:
                nums[u],nums[j]=nums[j],nums[u]
                j-=1
            else:
                u+=1
        